content/community/jdev-rules.md

```diff
@@ -0,0 +1,83 @@
+
+## Code of Conduct
+
+Like any growing community, the XMPP Developers Channel has a set of rules to govern the interactions, in order to make as many people as possible feel safe and welcome.
+
+Above all, the [Community Code of Conduct of the XMPP Standards Foundation](https://xmpp.org/extensions/xep-0458.html) (still under discussion, but unless stated otherwise in *this* document, the current experimental version applies nontheless) applies, called CoC hereafter. That means that behaviour which goes against the CoC is automatically not accepted.
+
+For transgressions solely happening within the XMPP Developers channel, the channel admins are an acceptable first point of contact.
+
+**Note**: The CoC only applies to XMPP Standards Foundation (XSF) members and venues. It does not automatically apply to all members of all XMPP channels. Do not report misconduct in venues outside of the control of the XSF to this room or the XSF Conduct Team, unless an XSF member is involved.
+
+## On-Topicness
+
+The channel started out as a place for developers to work together to improve the state of XMPP software in general, encompassing topics such as software interoperability (as opposed to deployment interoperability; for the latter, the [Operators channel](operators-rules.md) is the better a place), reliability, security and standards edge cases. It can provide valuable information when just reading passively, but also when actively asking.
+
+In order to keep the signal-to-noise ratio useful for people listening in on the channel passively, it is helpful to keep off-topic discussions to a minimum. To give you an idea about what people expect from the channel, here is a non-exhaustive list of topics which fit into the scope of the channel:
+
+* Discussion about tools for software developers specific to XMPP (e.g. XMPP test suites, test vectors, network simulation tools, XML linting, parsing, generation tools).
+* Reaching out to implementors of different software to discuss interoperability issues or find a place to responsibly disclose security issues.
+* Discussion of past security vulnerabilities in XMPP software.
+* Discussion of the effects of implementation choices on the broader ecosystem.
+* Asking questions about how specific pieces of a standard are meant to be interpreted.
+
+Please note the FAQ for a suggestion for places to discuss specific adjacent topics!
+
+### On-Topic induced Off-Topic
+
+Since some of the on-topic cases can quickly derail into off-topic when debates become ad-hominem. When debating controversial topics, the difference between an insult or judgement and a discussion is in the rationale.
```

Comment:
First sentence sounds incomplete to me. Maybe better without starting with "Since"?